src/index.html

```diff
@@ -5,9 +5,56 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/style.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
     <h1>Product cards</h1>
+    <div data-qa="card" class="card">
+      <img src="../src/images/imac.jpeg"
+      alt="iamge-pro"
+      class="card__image"
+      >
+
+      <p class="card__name">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </p>
+
+      <p class="card__code">
+        Product code: 195434
+      </p>
+
+      <div class="card__review">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <p class="card__review-count">
+          Reviews: 5
+        </p>
+      </div>
+
+      <div class="card__price-wrapper">
+        <p class="card__price-tittle">
+          Price:
+        </p>
+        <p class="card__price-value">
+          $2,199
+        </p>
+      </div>
+      <a
```

Comment:
add an empty line between multiline siblings
```suggestion
      </div>
      
      <a
```

Reply:
<img width="662" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/70220667/247ed61b-ce44-4fb9-889d-af3768d4e5c1">